pkg/webui/components/file-input/file-input.styl

```diff
@@ -0,0 +1,61 @@
+// Copyright © 2019 The Things Network Foundation, The Things Industries B.V.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+@require '../button/button.styl'
+
+.container
+  display: flex
+  align-items: center
+  justify-items: flex-start
+  margin-bottom: $cs.xxs
+
+.input
+  border: 0
+  clip: rect(0, 0, 0, 0)
```

Comment:
it makes sense to move this set of styles for visually hiding dom elements to a separate mixin. see https://allyjs.io/tutorials/hiding-elements.html#2017-edition-of-visuallyhidden for complete accessible solution